website/content/en/docs/deployment/_index.md

```diff
@@ -3,3 +3,12 @@ title = "Deployment"
 description = "Deploy Kubeflow on AWS"
 weight = 10
 +++
+Kubeflow on AWS is differentiated from the open-source KubeFlow project by a custom build manifest that provides integrations with various AWS services that are highly available and scalable. This reduces the operational overhead of maintaining the Kubeflow platform. 
```

Comment:
can we use something else rather than custom build manifest 

maybe by providing its own Kubeflow manifest?